src/utils/fetchMarkdown.ts

```diff
@@ -0,0 +1,18 @@
+interface MarkdownResult {
+	markdown: string;
+	__previousSource: string;
+}
+
+const fetchMarkdown = async (source: string): Promise<MarkdownResult> => {
+	let url = require(`../articles/${source}`);
+
+	const response = await fetch(url);
+	const newText = await response.text();
+	return {
+		markdown: newText,
+		__previousSource: source,
```

Comment:
`__previousSource` doesn't really make any sense in this context. Is there anything preventing us from moving this out?